packages/components/link/src/link.js

```diff
@@ -8,22 +8,49 @@ import { FormattedMessage } from 'react-intl';
 import { customProperties as vars } from '@commercetools-uikit/design-system';
 import { filterInvalidAttributes } from '@commercetools-uikit/utils';
 
+export const tones = {
+  primary: 'primary',
+  inverted: 'inverted',
+};
+const linkStates = {
+  normal: 'normal',
+  hover: 'hover',
+};
+
+export const whiteColor = '#FFFFFF';
+
 const getLinkStyles = (props, theme) => {
   const overwrittenVars = {
     ...vars,
     ...theme,
   };
 
+  const getColorValue = (toneProp, state) => {
+    const tone = tones[toneProp] ? toneProp : tones.primary;
+    const colorMap = {
+      [tones.primary]: {
+        [linkStates.normal]: overwrittenVars.colorPrimary,
+        [linkStates.hover]: overwrittenVars.colorPrimary25,
+      },
+
+      [tones.inverted]: {
+        [linkStates.normal]: whiteColor,
+        [linkStates.hover]: whiteColor,
+      },
+    };
+    return colorMap[tone][state];
+  };
```

Comment:
Hmm this looks overly complicated. We can also have two functions `getColorValue` and `getActiveColorValue`.